ironic/drivers/modules/drac/utils.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2017 Red Hat, Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
```

Comment:
Can you replace this header with the one we use for JetStream?  The copyright date should be 2020.
See: https://github.com/dsp-jetpack/JetPack/blob/master/src/pilot/customize_image.sh

Reply:
Addressed.